x-pack/plugins/security_solution/public/detections/components/alerts_table/alerts_utility_bar/index.test.tsx

```diff
@@ -17,17 +17,19 @@ describe('AlertsUtilityBar', () => {
   test('renders correctly', () => {
     const wrapper = shallow(
       <AlertsUtilityBar
-        hasIndexWrite={true}
-        hasIndexMaintenance={true}
         areEventsLoading={false}
         clearSelection={jest.fn()}
-        totalCount={100}
-        selectedEventIds={{}}
         currentFilter="closed"
+        hasIndexMaintenance={true}
```

Comment:
I have alphabetized a couple of props, which has created a suboptimal reviewer experience - my apologies for that. the only changes are the addition of a `showThreatMatchesOnly` boolean and a `onShowThreatMatchesOnlyChanged` function. In the future, I can refrain from alphabetizing, although I think it's easier to find stuff when everything is ordered, so either way.